mkosi

```diff
@@ -4339,6 +4349,10 @@ def load_args(args: CommandLineArguments) -> CommandLineArguments:
 
     args.extra_search_paths = expand_paths(args.extra_search_paths)
 
+    if args.selinux:
+        args.kernel_command_line.remove('selinux=0')
+        args.kernel_command_line.remove('audit=0')
```

Comment:
Let's invert this, remove these args from the default kernel args and if selinux is not specified, add them to the kernel args.